frontend/apps/reader/readerui.lua

```diff
@@ -14,6 +14,7 @@ local DocumentRegistry = require("document/documentregistry")
 local Event = require("ui/event")
 local FileManagerBookInfo = require("apps/filemanager/filemanagerbookinfo")
 local FileManagerCollection = require("apps/filemanager/filemanagercollection")
+local FileManagerDocument = require("document/filemanagerdocument")
```

Comment:
I don't like that name (file and variable, and file in document/ but named as the other FileManagerModules in apps/filemanager/).
No good other idea though.